### DISCUSSION TITLE
stepper events

### DISCUSSION THREAD

**Discussion by dfstefan** [mdbootstrap/TW-Elements#2131] [Support from community] [answered]
Hello, i am trying to use stepper component.
regardless the initialization method - js or with data- attributes i am unable to make it listen to the events stated in documentation - 
 onChangeStep.te.stepper 
 onValid.te.stepper
 onInvalid.te.stepper
also, there is something else wrong in documentation, as the event is *.te.stepper, and the documentation gives as example 

const stepOne = document.querySelectorAll('[data-te-stepper-step-ref]')[0];
stepOne.addEventListener('onChangeStep.mdb.stepper', () => {
  // do something...
});

any help with this is very appreciated. 
Thank you

**Comment by juujisai** [comment, marked as answer]
Hello, sorry to say that but something went wrong inside the stepper component. The events will not work for now. We will try to fix this issue ASAP and we will add more functionalities with next releases.

I've created a new issue for the events here:
https://github.com/mdbootstrap/TW-Elements/issues/2132
